__tests__/app.test.js

```diff
@@ -195,3 +195,82 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("POST /api/articles/:article_id/comments", () => {
+  test("responds with the posted comment", () => {
+    const comment = { username: "butter_bridge", body: "Hello world!" };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(comment)
+      .expect(201)
+      .then(({ body }) => {
+        const postedComment = body.comment;
+
+        expect(typeof postedComment.comment_id).toBe("number");
+        expect(typeof postedComment.body).toBe("string");
+        expect(typeof postedComment.article_id).toBe("number");
+        expect(typeof postedComment.author).toBe("string");
+        expect(typeof postedComment.votes).toBe("number");
+        expect(typeof postedComment.created_at).toBe("string");
+      });
+  });
+  test("responds with status 400 and 'bad request' if username does not exist in database", () => {
+    const comment = { username: "banana", body: "Hello world!" };
+    return request(app)
```

Comment:
This should be a 404, as the username is technically valid it just doesn't exist on the users table.